doc/nrf/releases_and_maturity/abi_compatibility.rst

```diff
@@ -47,57 +47,41 @@ The following table illustrates ABI compatibility between different versions of
 ABI compatibility ensures that the Secure Domain and System Controller firmware binaries do not need to be recompiled each time the application, radio binaries, or both are recompiled, as long as they are based on a compatible |NCS| version.
 Additionally, maintaining ABI compatibility allows the nRF54H20 SoC binary components to work together without recompilation when updating to newer |NCS| versions.
 
-nRF54H20 SoC Binaries v0.8.0 changelog
+nRF54H20 SoC Binaries v0.9.0 changelog
 **************************************
 
 The following sections provide detailed lists of changes by component.
 
-Secure Domain Firmware (SDFW) v9.0.0
+Secure Domain Firmware (SDFW) v10.0.0
 =====================================
 
 Added
 -----
 
-* Purge protection can be enabled over ADAC.
-* Clock control is enabled in SDFW.
-* Global domain power request service is integrated in SDFW.
-* PUF values from SDROM are cleared on boot.
+* SUIT in-place updatable components (IPUC) are now available to local domains over IPC through the SSF IPUC service.
+* GPIO DRIVECTRL for P6 and P7 on nRF54H20 is now corrected by SDFW on boot.
+* Added support for TLS-1.3.
+* Enable ED25519 pre-hashed.
 
 Updated
 -------
 
-* Local domain reset will trigger a global reset.
-  ``RESETINFO`` will contain both the global and local reset reason.
-* All processors are booted regardless of whether they have firmware.
-  They are booted in halted mode if no firmware is present.
```

Comment:
I think some of these changes would still be very noticeable to someone not having used the 0.8.0 version - do we want to remove them entirely?

Reply:
I agree, would an alternative be to not remove entries from old bundles? IMO all changes from all publicly available bundles should be listed here.